tests/Microsoft.Bot.Schema.Tests/CardExTest.cs

```diff
@@ -0,0 +1,259 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using Microsoft.Bot.Builder;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
```

Comment:
There is a parallel effort that is migrating all existing tests to XUnit. Let's not add new MSTests. Instead, please create these using XUnit.

Reply:
Changes applied. Thanks!